### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by jhgould**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Migration 
- [ ] Styling
- [ ] Testing
-[x] Documentation

### Commit Emojis:
:tada: Initial commit
:construction: HTML modifications/updates.
:art: CSS modifications/updates.
:crystal_ball: JS modifications/updates.
:lipstick: hotfix any code.
:bar_chart: new tests
:camera_flash: adding screenshots
:bento: Adding new assets or images
:beetle: If it is a bug or issue fix, describe the issue that was fixed.

### Detailed Description:
Updated Readme with the documentation for the endpoints. 
### Files modified:
(List all modified files here, making sure to check it. Please delete this line before commiting the PR!!)
- [ readme.md] 
- [ ] 
- [ ] 
- [ ]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,257 @@
-# to-the-moon-be
\ No newline at end of file
+# To the Moon and Backend 
+
+This is the final project at Turing School of Software and Design in Denver, CO. It is the backend portion of a service-oriented architecture. Working directly with a front end team, we were provided a detailed wireframe and a list of expected responses. We built an API that consumes multiple external APIs to gather and format that data required. 
+Stack: Python3, Flask, Postgres, Heroku 
+
+# Endpoints 
+
+### Celestial Bodies Index Endpoint
```
Comment (kiverso) [inline README.md:8]: We could probably include fewer celestial bodies and just use '. . . ' to signify that there would be more to avoid clutter.

Comment (jhgould) [reply README.md:8]: Good call. Ill make that update and create a new PR.

**Comment by jhgould** [comment]
I have updated the README and removed unnecessary information.

**Review by kiverso — APPROVED** [review]
Looks good

**Review by arpariseau — APPROVED** [review]

**Pull request merged**